pkg/manifests/types.go

```diff
@@ -691,6 +703,12 @@ type ThanosRulerConfig struct {
 	// with additional Alertmanager instances.
 	// The default value is `nil`.
 	AlertmanagersConfigs []AdditionalAlertmanagerConfig `json:"additionalAlertmanagerConfigs,omitempty"`
+	// Configures the default interval between Prometheus rule evaluations.
+	// The interval must be set between 5 seconds and 5 minutes.
+	// The value can be expressed in:
+	// seconds (for example `30s`.), minutes (for example `1m`.) or a mix of minutes and seconds (for example `1m30s`.).
+	// The default value is `30s`.
```

Comment:
```suggestion
	// The default value is `15s`.
```

Reply:
I couldn't find the default set in CMO but in operator we use `30s` https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/apis/monitoring/v1/prometheus_types.go#L1048, Am I missing something?

Reply:
aah my bad for Thanos its 15s https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/apis/monitoring/v1/thanos_types.go#L191